pkg/monitor/monitor/monitor_manager.go

```diff
@@ -105,6 +105,20 @@ func (m *MonitorManager) SyncMonitor(monitor *v1alpha1.TidbMonitor) error {
 		}
 	}
 
+	var firstDc *v1alpha1.DMCluster
+	if monitor.Spec.DM != nil {
+		for _, dcRef := range monitor.Spec.DM.Clusters {
```

Comment:
Do we need to log a warning if there are multiple clusters specified? Because we ignore some clusters.

Reply:
A warning is good.

Reply:
For the multiple cluster support, we assume that the versions of different TC/DC are the same so that we can use any of them to build the config of the monitor.
If they are different versions, the Grafana dashboards may be different, we do not support that case.
So I think no need to add a warning here.